content/Assets/Styles/base/_table.scss

```diff
@@ -0,0 +1,76 @@
+/* ==========================================================================
+   #TABLE
+   ========================================================================== */
+
+/**
+ * 1. The border-collapse property sets whether cells inside a <table> have
+ *    shared or separate borders. 
+ *    https://developer.mozilla.org/en-US/docs/Web/CSS/border-collapse
+ */
+
+table {  
+    width: 100%;
+
+    border-collapse: collapse; /* 1. */
+
+    thead {
+        display: none;
+
+        @include bp-min-width($bp-tablet) {
+            display: table-header-group;
+        }
+    }
+
+    tr {
+        display: block;
+        @include rem(margin-bottom, $spacing-default);
+        @include rem(padding-bottom, $spacing-default);
+
+        @include themify($themes) {
+            border-bottom: $table-border-width solid themed('color.border');
+        }
+
+        @include bp-min-width($bp-tablet) {
+            display: table-row;
+            margin-bottom: 0;
+        }
+    }
+
+    th,
+    td {
+        display: block;
+        @include rem(padding-bottom, $table-cell-padding-vertical-mobile);
+        @include rem(padding-left, $table-cell-padding-horizontal);
+        @include rem(padding-right, $table-cell-padding-horizontal);
+        @include rem(padding-top, $table-cell-padding-vertical-mobile);
+
+        text-align: right;
+
+        @include bp-min-width($bp-tablet) {
+            display: table-cell;
+            @include rem(padding-bottom, $table-cell-padding-vertical);
+            padding-right: 0;
+            @include rem(padding-top, $table-cell-padding-vertical);
+            vertical-align: top;
+
+            text-align: left;
+        }
+
+        &:last-child {
+            @include rem(padding-right, $table-cell-padding-horizontal);
```

Comment:
Is it just me or is this line redundant? Or should this last child bit be nested inside the bp-min-width mixin above?

Reply:
Agreed, looks redundant as the default `td` has the same padding.

Reply:
Tell a lie, it is needed otherwise the last column buts up right against the edge of the table.